datadog/dogstatsd/base.py

```diff
@@ -328,8 +345,9 @@ def __init__(
 
         # Check for deprecated option
         if max_buffer_size is not None:
-            log.warning("The parameter max_buffer_size is now deprecated and is not used anymore")
-
+            log.warning(
```

Comment:
I think your linter may have [ignored the config](https://github.com/DataDog/datadogpy/blob/master/tox.ini#L49). it looks like this got line truncated, but our line limit is 120 chars. 

Can you re-run the linter suggestions with the config applied? It will remove several unnecessary changes in this diff.